app/src/main/java/org/glucosio/android/tools/GlucoseRanges.java

```diff
@@ -53,56 +61,57 @@ void setPreferredRange(String preferredRange) {
 
     @VisibleForTesting
     void setCustomMin(int customMin) {
-        this.customMin = customMin;
+        this.userMin = customMin;
     }
 
     @VisibleForTesting
     void setCustomMax(int customMax) {
-        this.customMax = customMax;
+        this.userMax = customMax;
+    }
+
+    public static int getPresetMin(String preset) {
+        switch (preset) {
+            case ADA:
+                return ADA_MIN;
+            case AACE:
+                return AACE_MIN;
+            case UKNICE:
+                return UKNICE_MIN;
+            default:
+                return ADA_MIN;
```

Comment:
Why ADA is selected as default?

Reply:
It seemed reasonable. The default is only returned when the preset string given is not a known preset. It may be preferable to raise an error instead.

Reply:
Oke, clear